README.md

```diff
@@ -1,12 +1,46 @@
 # Self-healing Apache Zookeeper ECS cluster
 
 ## General information
-This repo contains terraform code and docker file which will allow you to create self-healing Apache Zookeeper cluster into AWS ECS
+The repository is a fork from https://github.com/dethmix/self-healing-zookeeper which the goal is to have a Zookeeper server that is self-healing and is deployed in AWS ECS, this has been further improved by removing the ECS autodiscovery feature as it creates complexity during deployment and automate the Zookeeper node enrollment.
 
 ## Usage
 Terraform code can be found in `terraform` directory, just update variable.tf file with values related to your AWS VPC and execute `terraform plan && terraform apply`
- 
-Dockerfile for building custom Apache Zookeeper image can be found in `docker` directory. It contains only few custom changes in comparing to the original Apache Zookeeper dockerfile. The main one is generation of `myid` from ECS service name.
 
-## Blog post
-More details about this setup can be found at https://tech.smartling.com/self-healing-apache-zookeeper-cluster-470b248ccb12
+
+## Infrastructure
+
+The infrastructure required to implement this solution is an EC2 container instance with a minimum of 3 subnets in which each subnet is equivalent to one container. 
```

Comment:
The infrastructure required to implement this solution are the following:

- Number of subnets equal to the number of nodes desired to be in the cluster
Note: Each subnet houses a single instance of a cluster.

- When the container comes up in the respective subnets, there are 2 requirements for that container to be part of the zookeeper cluster.  
             - The container needs to update its IP with correct domain in route53. We have provided that intelligence to the container where they register the domain appropriately with route53.
             -  The container needs to assign the correct zookeeper id to itself to be part of the zookeeper cluster